ClientSupport/src/main/java/io/deephaven/clientsupport/gotorow/SeekRow.java

```diff
@@ -46,37 +49,38 @@ public SeekRow(long startingRow, String columnName, Object seekValue, boolean in
     }
 
     @Override
-    @ConcurrentMethod
```

Comment:
Removing this isn't sufficient to make it not try to read while writes take place - the call to SeekRow.apply needs to be wrapped in a UGP shared lock, or we need to update SeekRow.apply's behavior to snapshot the table as we perform this operation (and restore the removed annotation). Reach out if you want a hand with either approach.